config/menu_links_with_sublinks_deletion.settings.yml

```diff
@@ -0,0 +1,2 @@
+enabled_menus:
+  gho-menu: gho-menu
```

Comment:
I think this needs a corresponding schema and a default config as part of the ghi_menu module.